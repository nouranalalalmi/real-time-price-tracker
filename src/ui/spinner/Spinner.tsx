export const Spinner = () => (
  <div
    data-testid="spinner"
    className="size-4 animate-spin rounded-full border-y-2 border-pink-500"
  />
);
