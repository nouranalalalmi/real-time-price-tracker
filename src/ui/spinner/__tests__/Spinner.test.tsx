import { render } from '@testing-library/react';

import { Spinner } from '../Spinner';

describe('Spinner', () => {
  it('should render the spinner', () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
