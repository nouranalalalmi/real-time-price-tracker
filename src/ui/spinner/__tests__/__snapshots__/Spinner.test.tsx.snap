// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render the spinner 1`] = `
<div>
  <div
    class="size-4 animate-spin rounded-full border-y-2 border-pink-500"
    data-testid="spinner"
  />
</div>
`;
